bgs/bgs.go

```diff
@@ -1017,9 +1024,12 @@ func (s *BGS) createExternalUser(ctx context.Context, did string) (*models.Actor
 
 	// TODO: request this users info from their server to fill out our data...
 	u := User{
-		Handle: handle,
-		Did:    did,
-		PDS:    peering.ID,
+		Did:         did,
+		PDS:         peering.ID,
+		ValidHandle: validHandle,
+	}
+	if validHandle {
+		u.Handle = handle
```

Comment:
i think we have a uniqueness constraint on handle in the db so we will have to explicitly set it to NULL

Reply:
Ah oops, yeah that's not a pointer, good call

Reply:
How do we do that with a `Create` using GORM?

Reply:
Might need to update that model to use a sql.NullString I think

Reply:
I would just manually write some SQL for this case. I hate using orm types for this, makes other things more annoying.

Reply:
ah, i see the change already. Thats not terrible i guess